### PR TITLE
Pal 232 jvm performance tests

### PR DESCRIPTION
**Pull request by dev930018**

### CODE REVIEW

File: readers-common/src/main/java/uk/gov/gchq/palisade/reader/common/SerialisingResponseWriter.java [hunk 1]
```diff
@@ -112,7 +112,7 @@ public ResponseWriter write(final OutputStream output) throws IOException {
             } else {
                 LOGGER.debug("Applying rules: {}", rules);
                 LOGGER.debug("Using serialiser {}", serialiser.getClass());
-                Stream<T> deserialisedStream = serialiser.deserialise(stream).peek(obj -> LOGGER.info("Deserialised object of type {} :: {}", obj.getClass(), obj));
+                Stream<T> deserialisedStream = serialiser.deserialise(stream);
```
Comment (developer01189998819991197253) [inline readers-common/src/main/java/uk/gov/gchq/palisade/reader/common/SerialisingResponseWriter.java:115]: do we not care about logging this info?

Comment (pd104923) [reply readers-common/src/main/java/uk/gov/gchq/palisade/reader/common/SerialisingResponseWriter.java:115]: Would it be better to have it at a debug level?

Comment (dev930018) [reply readers-common/src/main/java/uk/gov/gchq/palisade/reader/common/SerialisingResponseWriter.java:115]: It logs every single object read from the files.
At worst, the speed grinds to a halt and produces gigabytes of logging output, at best it still seems like a bit of a security flaw dumping out this data with no rules applied (ie no redactions)

Comment (ac74475) [reply readers-common/src/main/java/uk/gov/gchq/palisade/reader/common/SerialisingResponseWriter.java:115]: I agree with @dev930018 this is something that shouldn't really have made it into develop. It should only have been added for debugging and then removed

**Pull request merged**